generate.pl

```diff
@@ -41,8 +41,8 @@ sub die_with_sample {
 };
 
 my %builds = (
-  "64bit"          => "-Duse64bitall $common",
-  "64bit,threaded" => "-Dusethreads -Duse64bitall $common",
+  "64bit"          => "$common",
+  "64bit,threaded" => "-Dusethreads $common",
```

Comment:
I suppose we ought to change the `64bit` names here now to something else, since we can build 32- or 64-bit depending on `dpkg-architecture`.